app/controllers/container_controller.rb

```diff
@@ -58,7 +59,25 @@ def show
     redirect_to :action => 'show', :controller => record.class.base_model.to_s.underscore, :id => record.id
   end
 
+  def show_timeline
+    @showtype = "timeline"
+    session[:tl_record_id] = params[:id] if params[:id]
+    @record = Container.find_by_id(from_cid(params[:id]))
+    @timeline = @timeline_filter = true
+    @lastaction = "show_timeline"
+    tl_build_timeline                       # Create the timeline report
+    @refresh_partial = "layouts/tl_show"
+    if params[:refresh]
+      @sb[:action] = "timeline"
+      replace_right_cell
+    end
+  end
+
   def explorer
+    @display = params[:display] || "main" unless control_selected?
+    if @display == "timeline"
```

Comment:
this block can be changed to single line show_timeline if @display == "timeline"